src/sidebar/components/sidebar-content.js

```diff
@@ -32,7 +32,7 @@ function SidebarContent({
   // Store state values
   const focusedGroupId = useStore(store => store.focusedGroupId());
   const hasAppliedFilter = useStore(store => store.hasAppliedFilter());
-  const isFocusedMode = useStore(store => store.focusModeEnabled());
+  const isFocusedMode = useStore(store => store.focusModeConfigured());
```

Comment:
This reference `isFocusedMode` could probably have a clearer name.

Reply:
`showFocusModeHeader`?